sips/sip-010/sip-010-fungible-token-standard.md

```diff
@@ -0,0 +1,132 @@
+## Preamble
+
+- Sip Number: 010
+- Title: Standard Trait Definition for Non-Fungibe Tokens
+- Author: Hank Stoever (hstove@gmail.com)
+- Consideration: Technical
+- Type: Standard
+- Status: Draft
+- Created: 2021-01-25
+- License: CC0-1.0
+- Sign-off:
```

Comment:
Formatting -- can you add a `Discussions-To: https://github.com/stacksgov/sips` line?